packages/server/src/ws-server.ts

```diff
@@ -0,0 +1,20 @@
+import { WebSocketServer } from 'ws'
+
+import { AnyModels } from './types'
+import { connectWebSocket, createWSServerContext } from './ws-connection'
+
+export function createWSServer(models: AnyModels, port: number) {
+  const wss = new WebSocketServer({ port })
+
```

Comment:
nit: looking at the PR overall I like your concise style with no white spaces, so I would be down getting rid of them in other files like this one too! not a big deal tho!